jenkins-pipelines/openshift/Jenkinsfile

```diff
@@ -62,8 +64,7 @@ node ('openstack-jenkins-slave') {
 
     stage('validate openshift deployment') {
         sh("git clone -b $openshiftDeploymentAnsibleBranch https://github.com/UKCloud/openshift-deployment-ansible.git openshift-deployment-ansible")
-        sh("ansible-playbook ./openshift-deployment-ansible/tests/all.yml --extra-vars \"hostIP=$bastionip\" ")
-    }
+        sh("for var in \$(export | grep KUB | awk '{ print \$2 }' | sed 's/\\=.*//g'); do unset \$var; done; source ./openstack_rc.sh ; ansible-playbook ./openshift-deployment-ansible/tests/all.yml --extra-vars \"hostIP=$bastionip\" --extra-vars USER_USERNAME=\"\$USER_USERNAME\" --extra-vars USER_PASSWORD=\"\$USER_PASSWORD\" --extra-vars server=\"https://ocp.customer2.cor00005.cna.ukcloud.com:8443\"") }
```

Comment:
It would be great to add a comment describing what exactly we're doing here - that'll make it much more likely we clean it up and do it properly if we ever find a better way.